e2e/definitions/subject/deployment.yaml

```
@@ -22,6 +22,7 @@ spec:
         - -c
         - |
           printf '$BACKUP_FILE_CONTENT' | tee '/data/$BACKUP_FILE_NAME' && \
+          echo && \
```

would adding newline to printf also work?
```suggestion
          printf '$BACKUP_FILE_CONTENT' \n | tee '/data/$BACKUP_FILE_NAME' && \
```

---

scratch that, not worth changing :)

---

It would also change the behaviour. All of a sudden the `${BACKUP_FILE_CONTENT}` that is written to the file is actually `${BACKUP_FILE_CONTENT}\n`, which has to be compensated for in other places, like when comparing for equality. That was the point of using `printf '$BACKUP_FILE_CONTENT' in the first place. Otherwise I could have just used `echo '$BACKUP_FILE_CONTENT'`.